awscli/customizations/s3/filegenerator.py

```diff
@@ -151,8 +151,8 @@ def call(self, files):
 
     def _inject_extra_information(self, file_stat_kwargs, extra_information):
         src_type = file_stat_kwargs['src_type']
-        file_stat_kwargs['size'] = extra_information['Size']
-        file_stat_kwargs['last_update'] = extra_information['LastModified']
+        file_stat_kwargs['size'] = extra_information.get('Size')
```

Comment:
There is none. It does not matter if you provide `None` or omit them. No matter what one you chose, the end `size`  and `last_modified_time` properties on the `FileStat` object will be `None`